libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/JSONNetworkTests.m

```diff
@@ -1,62 +0,0 @@
-/*
```

Comment:
Removing this file as we dont have any tests here. At this point I dont see any value in keeping this around. We can add it in later as and when needed.